index.rst

```diff
@@ -0,0 +1,23 @@
+.. Mbed TLS documentation master file, created by
+   sphinx-quickstart on Thu Jul 29 17:27:29 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Mbed TLS's documentation!
```

Comment:
Maybe it's just me but I don't like this title. “Welcome to” is just noise: the title should start with the most important part, which is “Mbed TLS”. “Mbed TLS's” is awkward, “Mbed TLS documentation” would work just fine. I think there should be another qualifier there however to say that this is the central place for documentation, as opposed, say, to the documentation of a specific version. How about
```suggestion
Mbed TLS documentation hub
```